test-harness/specs/static_examples-AC-1.oas3.txt

```diff
@@ -0,0 +1,52 @@
+====test====
+Given
+
+Prism is started without --dynamic flag.
+
+When
+
+curl localhost:4010/todos
+
+Expect output to be
+
+{"title":"Work","description":"string","priority":0}
+====spec====
+info:
+  title: Sample API
+  version: 1.0.0
```

Comment:
You've been skipping this in the other documents, right? Maybe we can remove it from here too?